lib/lambda/sinkMainProcessors.ts

```diff
@@ -94,9 +94,8 @@ const getOneMacRecordWithAllProperties = (
     console.log(`event after transformation: ${JSON.stringify(oneMacRecord, null, 2)}`);
 
     return oneMacRecord;
-  } else {
-    console.log(`No transform found for event: ${record.event}`);
   }
+  console.log(`No transform found for event: ${record.event}`);
```

Comment:
I see you had to fix your original mistake that drove the whole if/else discussion. Good job!